### DISCUSSION TITLE
Is there any way to take GoPro Max's 360 video at 5~10 FPS?

### DISCUSSION THREAD

**Discussion by takerfume** [gopro/labs#317] [Feature Requests]
I would like to take take GoPro Max's 360 video at 5~10 FPS.

Is there any way to customize FPS?
Is there any firmware or some procedure?

**Comment by dnewman-gpsw** [comment]
That in not something that can be done in the stock camera, or in Labs.  TimeLapse fastest in 2fps, and the video modes slowest in 24fps. Only option today, down sampling in post.

**Reply by takerfume** [reply]
Okay.
Down sampling doesn't match my needs...
Anyway Thank you for your answer.
